learning_journal/templates/add_entry_template.jinja2

```diff
@@ -0,0 +1,9 @@
+{% extends "base.jinja2" %}
+{% block content %}
+<p>{{ error }}</p>
+<form method="POST">
+    <div>{{ form.title.label }}: {{ form.title() }}</div>
```

Comment:
does calling the title field of your form here actually work?  I've not seen that approach before.